akka-actor-typed-tests/src/test/scala/akka/actor/typed/OrElseSpec.scala

```diff
@@ -167,22 +167,22 @@ class OrElseSpec extends ScalaTestWithActorTestKit("""
 
     }
 
-
     "pass unhandled Terminated along" in {
       val probe = TestProbe[String]()
       spawn(Behaviors.setup[String] { ctx =>
-
         // arrange with a deathwatch triggering
         ctx.watch(ctx.spawnAnonymous(Behavior.stopped[String]))
 
-        Behaviors.receiveSignal[String] {
-          case (_, PreRestart) =>
-            Behaviors.same
-        }.orElse(Behaviors.receiveSignal {
-          case (_, Terminated(_)) =>
-            probe.ref ! "orElse saw it"
-            Behaviors.same
-        })
+        Behaviors
+          .receiveSignal[String] {
+            case (_, PreRestart) =>
+              Behaviors.same
+          }
+          .orElse(Behaviors.receiveSignal {
+            case (_, Terminated(_)) =>
+              probe.ref ! "orElse saw it"
+              Behaviors.same
+          })
```

Comment:
Scalafmt?

Reply:
Strange, would have expected that to be formatted in master already?

Reply:
Ya, very strange.

Reply:
PR validation was using `scalafmtCheck`.
I have changed to `scalafmtCheckAll` now.

Reply:
pro tip: enable scalafmt intellij plugin, use nightly, see instructions https://github.com/akka/akka/blob/master/CONTRIBUTING.md#scala-style

Reply:
Ah, I was doing the scalafmtAll, thanks.